### DISCUSSION TITLE
Welcome to Ditto Discussions!

### DISCUSSION THREAD

**Discussion by thjaeckle** [eclipse-ditto/ditto#1162] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tobias-zeptio** [comment]
So this is like a "Discussion" issue?

However, I work at Husqvarna and we are working on standardizing our IoT platform. A long and complicated process. Right now we are evaluating how to integrate Web of Things and ditto, if anyone have any success stories to share I'm happy to receive them. :)

**Reply by thjaeckle** [reply]
Hi @tobias-zeptio - thanks for the feedback, really appreciated 👍 
I thought that the WoT / Ditto integration is worth another top-level discussion, so I started #1163 - any feedback on that would be great.
